src/bundler.ts

```diff
@@ -132,7 +132,7 @@ class Bundler {
       return false;
     }
     if (targetBundle.url !== bundle.url) {
-      const relative = this._computeRelativeUrl(bundle.url, targetBundle.url);
+      const relative = urlUtils.relativeUrl(bundle.url, targetBundle.url);
```

Comment:
If bundle.url == targetBundle.url here then relative will be an empty string and will throw.  Can that case occur?